src/locale/locales/de/messages.po

```diff
@@ -558,28 +459,20 @@ msgstr "Eine E-Mail wurde an deine vorherige Adresse {0} gesendet. Sie enthält
 
 #: src/components/dialogs/GifSelect.tsx:252
 msgid "An error occured"
-msgstr ""
+msgstr "Es ist ein Fehler aufgetreten"
```

Comment:
```suggestion
msgstr "Ein Fehler ist aufgetreten"
```

both work but chatgpt says `Empfehlung: Für eine klare und direkte Kommunikation, insbesondere in technischen oder informellen Kontexten, klingt "Ein Fehler ist aufgetreten" besser.` and I agree